spectrocloud/resource_application_profile.go

```diff
@@ -62,6 +62,14 @@ func resourceApplicationProfile() *schema.Resource {
 					Type: schema.TypeString,
 				},
 			},
+			"tags_map": {
```

Comment:
@stefanSpectro If we need to fix in application profile as well ? then we need handle same in application profile crud operations